_projects/it/docs-italia.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Docs Italia
+subtitle: Documenti pubblici, digitali
+logo: /assets/images/docs-italia-logo.png
+payoff: Documenti pubblici, digitali
+description: "Docs Italia è lo strumento per la pubblicazione di documenti tecnici e amministrativi, offrendo ai cittadini la possibilità di leggere e commentare in modo semplice ed accessibile."
+lang: it
+order: 1
+highlight: false
+layout: project
+github_team: docs
+maintainers:
```

Comment:
Questi nickname devono fare riferimento al file _data/people.yml, ci sono tutti? Io sono alranel :-)

Reply:
Buona domanda, non ho approfondito perché mi pare che alla fine quei dati non siano visibili in pagina. L'ho popolata così perché su [altri progetti](https://raw.githubusercontent.com/italia/developers.italia.it/master/_projects/it/designers.md) è un array di nomi propri. Vedo un po' com'è usata e, nel caso, aggiorno.

Reply:
PS: grazie per l'hint, magari ne approfitto anche per aggiornare `_data/people.yml`.

Reply:
Aggiornato. Alla fine ti ho rimosso, in quanto ho visto che voi `admin` non avete assegnazioni sui singoli progetti. 🤷‍♂️